platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/oauth/OAuthSecurityImpl.java

```diff
@@ -360,6 +369,23 @@ private String getNewAccessToken(
     return accessToken;
   }
 
+  private void sanitizeFormParameters(Form formParam) {
+    MultivaluedMap<String, String> formParamMap = formParam.asMap();
+    List<String> passwords = formParamMap.get(PASSWORD);
+    if (passwords != null && passwords.size() == 1) {
+      String password = passwords.get(0);
+      if (isEncrypted(password)) {
+        LOGGER.warn("Trying to use an encrypted password to retrieve system access token.");
```

Comment:
This warning is still a little bit unclear but i am not sure how to make it clearer that the password should not be encrypted at this point.